docs/source/algorithms/ReflectometryILLPreprocess-v1.rst

```diff
@@ -11,8 +11,10 @@ Description
 
 This algorithm is the first step in the ILL reflectometry reduction workflow. It:
 
-#. loads data from disk
+#. loads data from disk by using LoadAndMerge <algm-LoadAndMerge> for :ref:`LoadILLReflectometry <algm-LoadILLReflectometry>` and :ref:`MergeRuns <algm-MergeRuns>`
```

Comment:
I think it is sufficient to mention `LoadAndMerge` only here.

Reply:
In my opinion, `LoadAndMerge` will not tell anything reflectometry reduction related to the user. If you visit the corresponding documentation, the relevant info for the user is null and an interested person would need to visit the code (which we like to avoid), maybe the info we get here is only that we load somehow and merge somehow. In particular, since we have merge options in the parameter file, we may want to be more specific here.